frontend/public/locales/en/upload.json

```diff
@@ -2,7 +2,7 @@
     "title": "Upload Knowledge",
     "subtitle": "Text, document, spreadsheet, presentation, audio, video, and URLs supported",
     "drop": "Drop the files here...",
-    "dragAndDrop": "Drag and drop files here, or click to browse",
+    "dragAndDrop": "Drag and drop files here, or click to browse \n (pdf, csv, Doc, Word, Txt, XIs)",
```

Comment:
You would need to update the other languages too - and using the same capitalization